src/Symfony/Component/Messenger/Transport/RedisExt/Connection.php

```diff
@@ -27,6 +27,13 @@
  */
 class Connection
 {
+    private const DEFAULT_OPTIONS = [
+        'stream' => 'messages',
+        'group' => 'symfony',
+        'consumer' => 'consumer',
+        'auto_setup' => true,
+    ];
+
```

Comment:
missing declarion of `private $autoSetup;`

Reply:
fixed